--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -725,6 +725,9 @@
       *mips32el*)
         tgt_isa=mips32
         ;;
+      riscv64*)
+        tgt_isa=riscv64
+        ;;
     esac
 
     # detect tgt_os
--- a/configure
+++ b/configure
@@ -159,6 +159,7 @@
 all_platforms="${all_platforms} x86_64-win64-vs12"
 all_platforms="${all_platforms} x86_64-win64-vs14"
 all_platforms="${all_platforms} x86_64-win64-vs15"
+all_platforms="${all_platforms} riscv64-linux-gcc"
 all_platforms="${all_platforms} generic-gnu"
 
 # all_targets is a list of all targets that can be configured
